### PR TITLE
Add history section to README

### LINKED ISSUE #11: LICENSE and History
I believe that the first commit of this project took heavy inspiration from https://github.com/cedadev/jasmin-dask/blob/master/docker/k8s-adaptive.py , which is currently unlicensed.  We should do a few things:

1.  Verify that we're not currently stealing content.  Are there still significant portions that are copied?
2.  Include a history giving that project and its authors credit, at least for inspiration

cc @yuvipanda @mkjpryor-stfc

### PR DESCRIPTION
**Pull request by mrocklin**
Fixes #11 

cc @yuvipanda @jacobtomlinson @mkjpryor-stfc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,3 +63,23 @@ process on one's local machine & the workers on minikube.
...
+[Informatics Lab](http://www.informaticslab.co.uk/).
+This Dask+Kubernetes solution was primarily developed by
+[Jacob Tomlinson](https://github.com/jacobtomlinson) of the Informatics Lab
+and [Matt Pryor](https://github.com/mkjpryor-stfc) of the
+[Science and Technology Facilities Council](http://www.stfc.ac.uk/).
+
+![Met Office Logo](https://raw.githubusercontent.com/met-office-lab/blog/master/img/mo-logo.svg)
+
+It was then adapted by [Yuvi Panda](http://words.yuvi.in/) at the UC Berkeley
+[BiDS](https://bids.berkeley.edu/) and [DSEP](http://words.yuvi.in/) programs
```
Comment (yuvipanda) [inline README.md]: second link is wrong, should probably be http://data.berkeley.edu/

**Comment by yuvipanda** [comment]
Minor nit, but lgtm otherwise! Thank you for writing this down :)

**Comment by mkjpryor-stfc** [comment]
@mrocklin: My work on this was funded by NERC (http://www.nerc.ac.uk). Any chance of getting a mention of that in my credit? Happy for STFC to be an acronym, as long as it is hyperlinked.

**Comment by jacobtomlinson** [comment]
cc @niallrobinson

**Pull request merged**